README.md

```diff
@@ -16,53 +16,51 @@
 
 </div>
 
-## Usage
-
-First, go to the repository you want to check.
+## Configuration
 
+This tool looks for configuration in a `pyproject.toml`file in the root of the codebase being analysed.
 
-### [black](https://black.readthedocs.io/en/stable/)
+The default configuration values are:
 
-Check **src** directory and **setup.py** file with **black**
-```bash
-docker run --rm -v "${PWD}":/github/workspace plone/code-quality:latest check black src setup.py
+```toml
+[tool.plone-code-analysis]
+checkers = ["black", "flake8", "isort", "pyroma", "zpretty"]
+formatters = ["black", "isort", "zpretty"]
+paths = "."
```

Comment:
I think the default should be `src docs setup.py` . "." it will probably not be useful in any situation as it will parse the `lib` folder. Only in `pyroma` the default should be ".". But this can be done in another PR.

Reply:
The idea was to always have some place to check -- as not every package has an src folder --, but we can discuss about that in another issue